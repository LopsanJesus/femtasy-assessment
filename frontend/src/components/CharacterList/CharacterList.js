import React from 'react';
import './CharacterList.scss';

const CharacterList = () => {
  return <div>CharacterList</div>;
};

export default CharacterList;
