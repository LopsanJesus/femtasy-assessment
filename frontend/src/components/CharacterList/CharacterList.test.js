import React from 'react';
import { shallow } from 'enzyme';
import CharacterList from './CharacterList';

describe('CharacterList', () => {
  test('matches snapshot', () => {
    const wrapper = shallow(<CharacterList />);
    expect(wrapper).toMatchSnapshot();
  });
});
