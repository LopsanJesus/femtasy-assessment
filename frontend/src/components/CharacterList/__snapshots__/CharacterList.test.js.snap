// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterList matches snapshot 1`] = `ShallowWrapper {}`;
